src/components/user/repo/user.repo.integration.test.js

```diff
@@ -88,4 +91,27 @@ describe('user repository', () => {
 			});
 		});
 	});
+
+	describe('createTombstoneUser', () => {
+		it('should return a user object', async () => {
+			const school = await testObjects.createTestSchool();
+
+			const tombstoneUser = await userRepo.createTombstoneUser(school._id, TOMBSTONE_SCHOOL_ID);
+
+			expect(tombstoneUser).to.be.an('object');
+			expect(tombstoneUser).to.contain.keys('email', 'firstName', 'lastName');
+		});
+
+		it('should return unique email adresses for different schools', async () => {
+			const numberOfTestSchools = 5;
+			const schools = await Promise.all([...Array(numberOfTestSchools)].map(() => testObjects.createTestSchool()));
+
+			const tombstoneUsers = await Promise.all(
+				schools.map((school) => userRepo.createTombstoneUser(school._id, TOMBSTONE_SCHOOL_ID))
+			);
+			const tombstoneEmailAdresses = tombstoneUsers.map((user) => user.email);
+
+			expect(new Set(tombstoneEmailAdresses).size).to.eql(5);
```

Comment:
Please check the e-mail format instead

Reply:
Should we really check the e-mail formats here? As far as I understand the test is supposed to checkt that the generated e-mail addresses are unique. This is implemented by a Set.
Or should we replace this test with one that checks if the generated e-mail format is correct?